### PR TITLE
Fix new qdeletion spam

### PR DESCRIPTION
**Pull request by Cyberboss**

### CODE REVIEW

File: code/controllers/subsystem/atoms.dm [hunk 1]
```diff
@@ -30,12 +30,16 @@ var/datum/controller/subsystem/atoms/SSatoms
...
 
+	var/static/list/NewQdelList = list()
+
 	if(atoms)
 		for(var/I in atoms)
 			var/atom/A = I
 			if(!A.initialized)	//this check is to make sure we don't call it twice on an object that was created in a previous Initialize call
 				if(QDELETED(A))
-					stack_trace("Found new qdeletion in type [A.type]!")
+					if(!(A.type in NewQdelList))
```
Comment (RemieRichards) [inline code/controllers/subsystem/atoms.dm]: ```if(!NeQdelList[A.type])```

File: code/controllers/subsystem/atoms.dm [hunk 2]
```diff
@@ -30,12 +30,16 @@ var/datum/controller/subsystem/atoms/SSatoms
...
+						stack_trace("Found new qdeletion in type [A.type]!")
+						NewQdelList += A.type
```
Comment (RemieRichards) [inline code/controllers/subsystem/atoms.dm]: ```NewQdelList [A.type] = TRUE```

File: code/controllers/subsystem/atoms.dm [hunk 3]
```diff
@@ -51,7 +55,9 @@ var/datum/controller/subsystem/atoms/SSatoms
 		for(var/atom/A in world)
 			if(!A.initialized)	//this check is to make sure we don't call it twice on an object that was created in a previous Initialize call
 				if(QDELETED(A))
-					stack_trace("Found new qdeletion in type [A.type]!")
+					if(!(A.type in NewQdelList))
```
Comment (RemieRichards) [inline code/controllers/subsystem/atoms.dm]: ```if(!NewQdelList[A.type])```

File: code/controllers/subsystem/atoms.dm [hunk 4]
```diff
@@ -51,7 +55,9 @@ var/datum/controller/subsystem/atoms/SSatoms
 		for(var/atom/A in world)
 			if(!A.initialized)	//this check is to make sure we don't call it twice on an object that was created in a previous Initialize call
 				if(QDELETED(A))
-					stack_trace("Found new qdeletion in type [A.type]!")
+					if(!(A.type in NewQdelList))
+						stack_trace("Found new qdeletion in type [A.type]!")
+						NewQdelList += A.type
```
Comment (RemieRichards) [inline code/controllers/subsystem/atoms.dm]: ```NewQdelList[A.type] = TRUE```

**Comment by AnturK** [comment]
Thanks.

**Comment by Poojawa** [comment]
Have implimented this.


[23:13:47] Runtime in unsorted.dm, line 1242: Found new qdeletion in type /obj/effect/landmark!
[23:13:47] Runtime in unsorted.dm, line 1242: Found new qdeletion in type /obj/machinery/vending/cola/random!
[23:13:47] Runtime in unsorted.dm, line 1242: Found new qdeletion in type /obj/effect/spawner/lootdrop/maintenance!
[23:13:48] Runtime in unsorted.dm, line 1242: Found new qdeletion in type /obj/effect/landmark/latejoin!
[23:13:48] Runtime in unsorted.dm, line 1242: Found new qdeletion in type /obj/machinery/computer/arcade!
[23:13:48] Runtime in unsorted.dm, line 1242: Found new qdeletion in type /obj/item/weapon/coin/gold!
didn't help none,

All of them report back with 
proc name: stack trace (/proc/stack_trace)

**Pull request merged**